docs/src/main/sphinx/connector/bigquery.md

```diff
@@ -430,8 +430,22 @@ FROM
 ```{include} query-table-function-ordering.fragment
 ```
 
+## Performance
+
+The connector includes a number of performance improvements, detailed in the
+following sections.
+
+(bigquery-pushdown)=
+### Pushdown
+
+The connector supports pushdown for a number of operations:
+
+- [](limit-pushdown)
```

Comment:
LIMIT pushdown in BigQuery connector is tricky. We should document the limitations. 
https://github.com/trinodb/trino/blob/b2dc49ae121c04a531944e09a433c7327fe24377/plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryMetadata.java#L1105-L1110

Reply:
I tried to translate this into use facing wording. Not ideal but better than nothing.